utils/test/context/context.util.tsx

```diff
@@ -36,4 +42,37 @@ const renderWithContextProviders: <T>(
     ...options,
   });
 
-export { renderWithContextProviders };
+/** renderWithConfigFeatureFlagContext renders the given `ui` JSX with a ConfigFeatureFlag provider seeded with default values. Use this if your components indirectly use/require a value from the ConfigFeatureFlag context, else use `renderWithCustomConfigFeatureFlagContext` so the value from ConfigFeatureFlag can be controlled  */
+const renderWithConfigFeatureFlagContext: (
+  ui: ReactElement,
+  options?: Record<string, unknown>
+) => RenderResult = (ui, options = {}) =>
+  renderWithContextProviders(ui, options, [
+    {
+      provider: ConfigFeatureFlag.Provider,
+      value: defaultConfigFeatureFlagValue,
+    },
+  ]);
+
+/** renderWithCustomConfigFeatureFlagContext renders the given `ui` JSX with a ConfigFeatureFlag provider provided via `configFeatureFlagValue`. */
+const renderWithCustomConfigFeatureFlagContext: (
```

Comment:
Can these be composed? I.e can I perform `renderWithConfigFeature(renderWithCustomConfig)` - or a builder esque pattern that allows you to do `withX().withY().withZ().render()`?

Reply:
Agree it would be nice to have something more fluent, but I was building upon what was already there. Follow on issue?